README.md

```diff
@@ -2,9 +2,72 @@
 
 An operator to deploy the [OpenStack Cinder CSI driver](https://github.com/openshift/cloud-provider-openstack/tree/master/pkg/csi/cinder) in OpenShift.
 
-# Quick start
+## Configuration
 
-Before running the operator manually, you must remove the operator installed by CSO/CVO
+The operator can be configured using a config map, which by default is found at either `openshift-config / cinder-csi-config` or `openshift-config / cloud-provider-config`.
+The former is preferred as it stores configuration solely for the Cinder CSI Driver.
+It is supported starting in OpenShift 4.14 and should be used in all new deployments.
+As the name might suggest, the latter stores configuration for both the Cinder CSI driver and the OpenStack Cloud Provider.
+It is used for legacy reasons.
+
+Configuration must be stored in the config map under the `config` key. For example:
+
+```shell
+oc get configmap -n openshift-config cinder-csi-config -o yaml
+```
+
+```yaml
+apiVersion: v1
+data:
+  ca-bundle.pem: |
+    <redacted>
+  config: |
+    [Global]
+    ...
+    [BlockStorage]
+    ...
+kind: ConfigMap
+metadata:
+  name: cinder-csi-config
+  namepsace: openshift-config
+```
+
+> *Note*
+> This config map stores configuration for both services for historical reasons: previously, block device management was handled by the cloud provider.
```

Comment:
This note seems to refer to the `cinder-csi-config` configmap. Does this mean that both the historical configmap (`cloud-provider-config`) and the new one (`cinder-csi-config`) store configuration for both services?